docs/cli/overview.mdx

```diff
@@ -45,19 +45,19 @@ The Infisical CLI provides a way to inject environment variables from the platfo
 	 <Tab title="Alpine">
 	 Install prerequisite
 		```bash
-		sudo apk add --no-cache bash sudo
+		apk add --no-cache bash curl
 		```
 
     	Add Infisical repository
     	```bash
     	curl -1sLf \
     		'https://dl.cloudsmith.io/public/infisical/infisical-cli/setup.alpine.sh' \
-    		| sudo -E bash
+    		| bash
     	```
 
     	Then install CLI
     	```bash
-    	sudo apk update && sudo apk add infisical
+    	apk update && apk add infisical
```

Comment:
for apk ad, it will also require sudo permission

Reply:
![Screenshot 2023-02-11 at 11 40 04 AM](https://user-images.githubusercontent.com/31166322/218243480-e6ca428f-3d76-46c5-921a-a7f8dbb2a300.png)

hey @maidul98 For both alpine and amazon linux throws error for sudo.

I think its clear in [docker](https://infisical.com/docs/integrations/platforms/docker) section.

For deb yah i realized it's quite dependent